src/app/public/modules/shared/initialization.service.ts

```diff
@@ -39,6 +49,7 @@ export class HelpInitializationService {
       const languages = skyuxHost.acceptLanguage || '';
       config.locale = languages.split(',')[0];
     }
-    return BBHelpClient.load(config);
+    const newConfig: Observable<HelpWidgetConfig> = this.extension ? this.extension.extend(config) : of(config);
+    newConfig.subscribe(c => BBHelpClient.load(c));
```

Comment:
the [existing use](https://github.com/blackbaud/skyux-sdk-builder/blob/master/src/app/app.component.ts#L296) of this wasn't actually using the promise that `BBHelpClient.load` returns. hence why i got rid of the return. but upon [further investigation](https://dev.azure.com/blackbaud/Products/_search?text=repo%3Askyux*%20HelpInitializationService%20load%20NOT%20file%3A%20app-extras.module.ts%20NOT%20file%3A*spec*%20NOT%20path%3A%2F.skypagestmp%2F*&type=code&pageSize=25&filters=ProjectFilters%7BProducts%7D&action=contents&result=DefaultCollection%2FProducts%2Fskyux-lib-sis-common%2FGBmaster%2F%2Fsrc%2Fapp%2Fpublic%2Fsrc%2Fmodules%2Fsis-common-core%2Finit-help.component.ts), others are using this promise. so i'm going to tweak this so a promise is returned.

Reply:
also ... why did i use the variable `c`? that's embarrassing.